app/components/CanvasContainer.js

```diff
@@ -209,8 +187,10 @@ export default class CanvasContainer extends Component {
       <div>
         <Grid id="canvas-header">
           <Col lg={12}>
-          {this.renderEditButtons()}
-          {this.renderPageNavButtons()}
+            <ButtonToolbar className="page-and-view-buttons">
+              {this.renderEditButtons()}
```

Comment:
It's probably better practice to split these into their own dumb components instead of writing methods that return jsx chunks.